### PR TITLE
Schema

### PR DESCRIPTION
**Pull request by chinkevin25**
@connieyeah Please review my PR for postgres

### CODE REVIEW

File: db/generateData.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const generatePartySize = () => Math.floor(Math.random() * 19 + 1);
 let primaryKey = 1;
 const generateReservations = (restaurantID) => {
   const data = [];
-  for (let day = 0; day < 10; day += 1) {
+  for (let day = 0; day < 1; day += 1) {
```
Comment (connieyeah) [inline db/generateData.js]: What's the reason for this change?

**Review by connieyeah — APPROVED** [review]
Looks good to me

File: db/generateData.js (hunk 1 above)
Comment (chinkevin25) [reply db/generateData.js]: Thanks for catching this, I reduced the size of my data for testing purposes but forgot to revert this!

This value should be 10.
